### ISSUE TITLE
Some questions when using dropout

### ISSUE THREAD

**Issue by nian-liu** [nitishsrivastava/deepnet#41]
Hi Nitish,
  I am using dropout in deepnet now. But I got 2 questions about details of the usage of dropout. I am wondering could you or anyone else help me.
1. When I do layer-wise pretraining for a DBM   with dropout, for example, the DBM consist of a input layer, 2 hidden layers, h1 and h2, with dropout_prob 0.2, 0.5, 0.5 respectively. After I pretrained the bottom RBM(RBM1), is it correct that I directly use the extracted representations of RBM1 to train the top RBM(RBM2)? 
  What I mean is that, when extract the representations of RBM1, the nodes of h1 would be multiplied by (1-dropout_prob) after actived. When these data are used to train RBM2 as inputs, they would be corrupted again by being multiplied by the dropout mask, it seems incorrect in my mind.
2. I am training a multimodal DBM with dropout with 2 modalities. I will use the representations of the joint hidden layer as features to do regression by SVR. Having read that dropout should not be used in the output layer of a DL model , I am wondering in my case, should the joint hidden layer be seen as a output layer? Is it proper to use dropout in it? 
Thanks in advance.
